src/OrleansLoggerProvider/LegacyOrleansLoggerProvider.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.LegacyOrleansLoggerProvider</PackageId>
+    <Title>Microsoft Legacy Orleans Logger Provider</Title>
+    <Description>Library of LegacyOrleansLoggerProvider.</Description>
+    <PackageTags>$(PackageTags) Orleans</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>LegacyOrleansLoggerProvider</AssemblyName>
```

Comment:
Please exclude the redundant Orleans from the assembly and package name. And I'd remove Provider from there too, since we are probably going to add the Logger stuff too in the very near future.
Just call it LegacyLogging (and package name Microsoft.Orleans.LegacyLogging)

Reply:
at a second thought, I think we should just name it as Logging and Microsoft.Orleans.Logging. Since this assembly includes both legacy feature and non-legacy feature, such as event bulking. 

I added Orleans.Extensions.Logging.Legacy namespace to differentiate legacy feature and non-legacy ones

Reply:
But bulking itself we are only including for legacy reasons. This should not be an Orleans concern, but an application concern. If we hadn't had that feature before, we should not be including this here.

Reply:
Agree, see my first comment in the review.